src/main/java/at/hannibal2/skyhanni/features/commands/WikiCommand.kt

```diff
@@ -26,9 +26,12 @@ class WikiCommand {
                 val search = packet.message.substring(6)
                 LorenzUtils.chat("§e[SkyHanni] Searching the Fandom Wiki for §c$search")
 
-                val url = "https://www.google.com/search?q=inurl%3Ahypixel-skyblock.fandom.com $search&hl=en"
+                //val url = "https://www.google.com/search?q=inurl%3Ahypixel-skyblock.fandom.com $search&hl=en"
+                //val poggers = true
+                //val google = !poggers
+                val url = "https://hypixel-skyblock.fandom.com/wiki/Special:Search?query=$search&scope=internal"
```

Comment:
Maybe we could remove those comments (and also use a setting for this)

Reply:
hanni said, and i quote:
![image](https://github.com/hannibal002/SkyHanni/assets/51521765/b89f2056-eaa9-41da-a0d4-f60fe6381121)

to be fair it saves a lot of grief on choosing a search engine and instead goes straight into the horse's mouth when it comes to wiki searches

Reply:
also *wow* i need to turn my github notifications on more often i have missed a lot of comments from you